ocf_data_sampler/select/select_spatial_slice.py

```diff
@@ -45,9 +44,6 @@ def convert_coords_to_match_xarray(
         if from_coords == "osgb":
             x, y = osgb_to_geostationary_area_coords(x, y, da)
 
-        elif from_coords == "lon_lat":
-            x, y = lon_lat_to_geostationary_area_coords(x, y, da)
-
```

Comment:
potentially silly question, but isn't this needed for satellite data and lat-lon poi? or do we not have that in pvnet => chuck it for now?

Reply:
We don't use this in UK PVNet and we haven't brought across the `lon_lat_to_geostationary_area_coords()` function from ocf_datapipes yet. So maybe it is more in keeping with our minimalist approach for now